client/login/wp-login/style.scss

```
@@ -83,7 +83,7 @@ $image-height: 47px;
 	}
 }
 
-.wp-login__main.main {
+.layout:not(.woo) .wp-login__main.main {
```

I think this could be `,is-white-login` instead of `.woo`, but not sure (hopefully it is). I think other screens may suffer the same. Can we add the comment below ("clk" is just my initials):

```suggestion
/* TODO clk consider .is-white-login instead of .woo */
.layout:not(.woo) .wp-login__main.main {
```

---

I think you (or someone else) already removed this block of code in another PR, rebasing it got rid of it fully.